autoload/jsdoc.vim

```diff
@@ -111,7 +111,7 @@ function! jsdoc#insert()
 
   let l:line = getline('.')
   let l:indentCharSpace = ' '
-  let l:indentCharTab = '	'
+  let l:indentCharTab = '       '
```

Comment:
@heavenshell Whoups, I guess it was replaced by my VIM configs automatically. Fixed. :smile: